templates/word-daily-level/src/addons/words/grid.js

```diff
@@ -0,0 +1,107 @@
+import { BULL, COW, EMPTY, NONE } from "./keyState";
+import "./grid-style.css";
+
+// The NxM grid of letters
+export class Grid extends HTMLElement {
```

Comment:
What do you think of having a file naming convention around the custom elements? Eg: `grid-element.js`. It would make it easier to tell at a glance which files in a project drive the display and which are game logic.

We should also make the .css the same base name as the .js to hint that these files are meant to be paired.